Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Configurations/IDeviceSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using OpenQA.Selenium.Appium;
+
+namespace Aquality.Appium.Mobile.Configurations
+{
+    /// <summary>
+    /// Describes desired device settings.
+    /// </summary>
+    public interface IDeviceSettings
+    {
+        /// <summary>
+        /// Options (capabilities) related to desired device.
+        /// </summary>
+        AppiumOptions AppiumOptions { get; }
```

Comment:
@mialeska , pay attention to this name. I'm not sure that this is sufficient name for this property. In java library we have `getCapabilities()` method. Moreover, we also have such property in `IDriverSettings`. Does it make sense to unify name in both .NET and Java libraries?

Reply:
I believe no, because they represent different types: AppiumOptions in appium dotnet-client and Capabilities in appium java-client.
Are you arguing that property name should represent the sense of the object? Of course, we could name it "capabilities" in both cases, but this would be a lie :) Btw, we have ICapabilities in appium dotnet-client, it's quite a different type, although AppiumOptions are convertible to ICapabilities